metadata-transformer/src/main/java/no/unit/nva/metadata/Bibo.java

```diff
@@ -0,0 +1,21 @@
+package no.unit.nva.metadata;
+
+import org.eclipse.rdf4j.model.IRI;
+import org.eclipse.rdf4j.model.ValueFactory;
+
+public enum Bibo implements OntologyProperty {
```

Comment:
This enum approach may not seem entirely normal, I if you have concerns, raise them. 

Here, I am using the traditional property re-use strategy from the linked open data concept, I understand that this may be unusual, but the term bibo:doi is well defined and is lacking in dcterms.